views/artist-search-results.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="row row-cols-1 row-cols-md-3 g-4 p-5">        
+        {{#each results.artists.items}}
+            <div class="col">
+                <div class="card h-100 text-center">
+                    <img src="{{images.0.url}}" class="card-img-top" alt="{{name}}">
```

Comment:
¿y si no tiene imagen?, podrias poner condicionalmente ese src, en caso de que no exista poner una imagen por defecto, para que no queden resultados como el que te paso en esta captura. 

![image](https://user-images.githubusercontent.com/44052311/107554278-053d2a00-6bd6-11eb-8988-0694c51b21fe.png)

Te dejo enlace util por si no quieres hacerlo con handlebars, 
https://stackoverflow.com/questions/3984287/how-to-show-alternate-image-if-source-image-is-not-found-onerror-working-in-ie